From: Openkylin Developers <packaging@lists.openkylin.top>
Date: Thu, 8 Sep 2022 02:10:22 +0000
Subject: Drop build-dependency on libext4_utils

fastboot uses none of the symbols provided by libext4_utils but still includes
a header of it. Removing it shall be enough.
---
 fastboot/fs.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fastboot/fs.cpp b/fastboot/fs.cpp
index 709f061..8006beb 100644
--- a/fastboot/fs.cpp
+++ b/fastboot/fs.cpp
@@ -23,7 +23,6 @@
 #include <android-base/file.h>
 #include <android-base/stringprintf.h>
 #include <android-base/unique_fd.h>
-#include <ext4_utils/make_ext4fs.h>
 #include <sparse/sparse.h>
 
 using android::base::StringPrintf;
